JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -545,7 +545,7 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
   if(options[["explanatoryText"]]){
 
     distribution <- base::switch(options[["distribution"]], "benford" = "Benford's law", "uniform" = "the uniform distribution")
-    benfordsLawPlotText <- createJaspHtml(gettextf("<b>Figure %i:</b> The observed percentages of each number in the population compared to the expected percentage under %2$s. The more the blue dots lie near the top of the grey bars, the more the population conforms to %2$s.", jaspResults[["figNumber"]]$object, distribution), "p")
+    benfordsLawPlotText <- createJaspHtml(gettextf("<b>Figure %i:</b> The observed percentages of each digit in the data set compared to the expected percentage under %2$s. The more the blue dots lie near the top of the grey bars, the more the data set conforms to %2$s.", jaspResults[["figNumber"]]$object, distribution), "p")
```

Comment:
"<b>Figure %i:</b> The observed percentages of each digit in the data set compared to the expected percentage under %2$s. For data sets distributed according to %2$s the blue dots will lie near the top of the grey bars."